dig.go

```diff
@@ -380,10 +381,8 @@ func getParameterObject(c *Container, t reflect.Type) (reflect.Value, error) {
 
 		v, err := c.get(f.Type)
 		if err != nil {
-			// TODO: once other tags are avilabled (named=foo),
-			// probably should serialize the tags better, or at least scan the string
-			tag := f.Tag.Get("dig")
-			if tag == "optional" {
+			tags := strings.Split(f.Tag.Get("dig"), ",")
+			if containsString(tags, "optional") {
```

Comment:
ah makes sense. I was thinking `"dig:optional"` or `"dig:Optional"` should work either way.